File: ClientApp/src/app/_models/LookupData.ts
export interface LookupData { 
    lookup_value_id?:number;
    lookup_type_id?:number;
    lookup_value_name:string;
    lookup_value_description:string;
    lookup_type_name:any;
    status_flag:boolean;
    lookup_type: string;
    Page: number;
    Size :number;
    SearchText : string;
    contact_id : string;
    ids:any;
    tax_module_id: number
    StatusFlag: number
    totalrows: number
    totalselectedrows: number
    misc_value: string
}

export interface MisclookupData { 
    lookup_value_id?:number;
    lookup_type_id?:number;
    lookup_value_name:string;
    lookup_value_description:string;
    lookup_type_name:any;
    status_flag:boolean;
    lookup_type: string;
    Page: number;
    Size :number;
    SearchText : string;
    contact_id : string;
    ids:any;
    tax_module_id: number
    StatusFlag: number
    totalrows: number
    totalselectedrows: number

}

export interface MessageData{
    messaging_event:string,
    message_event_type:string,
    messaging_event_description:string,
    messaging_event_id:number,
    tax_module_id: number
}

export interface GetMisslinousData{
    contact_id:number,
    Page:number,
    Size:number,
    SearchText?:string,
}

export interface GetMessageSave{
    ids:any;
    tax_module_id: number
}

File: OLDMutual.CTS.Admin.Domain/Models/LookupData.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace OLDMutual.CTS.Admin.Domain.Models
{
    public class LookupData
    {
        public int? lookup_value_id { get; set; }
        public int lookup_type_id { get; set; }
        public string lookup_value_name { get; set; }
        public string lookup_type_name { get; set; }
        public string lookup_value_description { get; set; }
        public bool status_flag { get; set; }
        public int totalrows { get; set; }
        public int totalselectedrows { get; set; }
        public string misc_value { get; set; }
    }
}
